addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -174,7 +173,7 @@ export class HtmlField extends Component {
         return this.props.record.fields[this.props.name].translate;
     }
     get markupValue () {
```

Comment:
if the method name still correct?

Reply:
the method is override by FieldPostPreview, and used on the xml.
I wasn't sure if the name is correct or not, but I decided to leave as is, I thought modifying the name was a little out-off scope of this PR.
If the @odoo/rd-framework-editor wants to change the name, I can do it in this PR.

Reply:
I thought this was about having `value` in the property name then I realized the call to `markup` was removed too. I read the commit message but it gave me no clue why this call was removed. I'm confused.

Reply:
Thanks for the clarification. So `markupValue` is still valid as a name for this value, it's just that it is kind of redundant now. I'd have argued to remove it but since the getter is indeed overriden as it seems, I think it's out of scope of this PR to do so. Thanks to everyone involved !